### PR TITLE
get default lidar updateRate from .proto

### PR DESCRIPTION
**Pull request by delihus**
**Description**
The default frequency of the lidar sensor is 30Hz when the `updateRate` is not defined in URDF.
This change gets the frequency from the webots lidar device and override default `updateRate`.

**Related Issues**
https://github.com/cyberbotics/webots/pull/5973
**Affected Packages**
List of affected packages:
  - webots_ros2_driver

### CODE REVIEW

**Review by ygoumaz — CHANGES_REQUESTED** [review]
Thank you! Please also update the [main changelog file](https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2/CHANGELOG.rst) and the one in the [webots_ros2_driver](https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2_driver/CHANGELOG.rst) package.

File: webots_ros2_driver/src/plugins/static/Ros2Lidar.cpp [hunk 1]
```diff
@@ -23,12 +23,15 @@ namespace webots_ros2_driver
 {
   void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters)
   {
+    mLidar = wb_robot_get_device(parameters["name"].c_str());
+    assert(mLidar != 0);
+    if(parameters.count("updateRate") == 0.0){
+      parameters.insert({"updateRate", std::to_string(wb_lidar_get_frequency(mLidar))});
+    }
```
Comment (omichel) [inline webots_ros2_driver/src/plugins/static/Ros2Lidar.cpp:30]: Coding style:
```suggestion
    if (parameters.count("updateRate") == 0)
      parameters.insert({"updateRate", std::to_string(wb_lidar_get_frequency(mLidar))});
```

File: webots_ros2_driver/src/plugins/static/Ros2Lidar.cpp [hunk 2]
```diff
@@ -19,14 +19,18 @@
 
 #include <webots/robot.h>
 
-namespace webots_ros2_driver {
-  void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters) {
+namespace webots_ros2_driver
+{
+  void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters)
+  {
```
Comment (omichel) [inline webots_ros2_driver/src/plugins/static/Ros2Lidar.cpp:25]: That should fix the clang-format issue:
```suggestion
namespace webots_ros2_driver {
  void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters) {
```

**Comment by omichel** [comment]
The tests are now passing.

**Review by ygoumaz — APPROVED** [review]
Thank you!

**Comment by omichel** [comment]
@delihus, I just sent you an invite to join the GitHub Webots committers group, this will ease your next contributions (no need to fork our repos, merge button enabled, etc.)

**Pull request merged**